renderer/reducers/settings.js

```diff
@@ -60,25 +76,23 @@ export const saveConfigOverrides = values => async (dispatch, getState) => {
 // Action Handlers
 // ------------------------------------
 const ACTION_HANDLERS = {
-  [SET_SETTINGS]: (state, { settings }) => ({
-    ...state,
-    ...settings.reduce((obj, item) => {
-      obj[item.key] = item.value
-      return obj
-    }, {}),
-  }),
-  [SET_SETTING]: (state, { key, value }) => ({
+  [INIT_SETTINGS]: state => ({ ...state }),
+  [INIT_SETTINGS_SUCCESS]: (state, { settings }) => ({
     ...state,
-    ...{ ...state.settings, [key]: value },
+    ...settings,
+    isSettingsLoaded: true,
   }),
+  [INIT_SETTINGS_FAILURE]: (state, { initSettingsError }) => ({ ...state, initSettingsError }),
```

Comment:
are init settings called only once per APP lifecycle? if not, then we need to clear error somewhere

Reply:
Yes, this is expected to be called once and once only, when the root component mounts.

Reply:
Have added a comment to note that.